deltachat-ios/Helper/NotificationManager.swift

```diff
@@ -19,11 +19,28 @@ public class NotificationManager {
                 if !UserDefaults.standard.bool(forKey: "notifications_disabled") && !chat.isMuted {
                     let content = UNMutableNotificationContent()
                     let msg = DcMsg(id: messageId)
-                    content.title = msg.getSenderName(msg.fromContact)
-                    content.body = msg.summary(chars: 40) ?? ""
+                    content.title = chat.isGroup ? chat.name : msg.getSenderName(msg.fromContact)
+                    content.body =  msg.summary(chars: 80) ?? ""
+                    content.subtitle = chat.isGroup ?  msg.getSenderName(msg.fromContact) : ""
                     content.userInfo = ui
                     content.sound = .default
 
+                    if msg.type == DC_MSG_IMAGE || msg.type == DC_MSG_GIF,
+                       let url = msg.fileURL {
+                        do {
+                            // make a copy of the file first since UNNotificationAttachment will move attached files into the attachment data store
```

Comment:
does that run in the ui-thread (that may get some anr then) or in some background-thread?

not sure what `OperationQueue.main` means here suggest also here to add a documentation line atop, even if everything is already fine.